static/js/containers/UserPage_test.js

```diff
@@ -41,11 +44,23 @@ describe("UserPage", () => {
 
   let userActions = [RECEIVE_GET_USER_PROFILE_SUCCESS, REQUEST_GET_USER_PROFILE];
 
+  let openDialog = () => {
+    return [...document.getElementsByClassName('deletion-confirmation')].find(dialog => (
+      dialog.style["left"] === "0px"
+    ));
+  };
+
   afterEach(() => {
     helper.cleanup();
   });
 
   describe("Education History", () => {
+    let deleteButton = div => {
+      return div.getElementsByClassName('profile-tab-card')[1].
+        getElementsByClassName('profile-row-icons')[0].
+        getElementsByClassName('mdl-button')[1];
```

Comment:
Can you use the CSS class for the delete button here, like you do for the cancel button below?